nano-layout.el

```diff
@@ -18,7 +18,7 @@
 
 (setq default-frame-alist
       (append (list
-	       '(font . "Roboto Mono:style=Light:size=14")
+	       '(font . "Roboto Mono:style=Light")
```

Comment:
Why did you removed size?

Reply:
Just read you answer. But I'm on HiDPI screen and 14 is big enough I would say.

Reply:
:thinking: On linux / gnome, this results in extra small text. Not setting it reverts to a sane default.

Reply:
See a side by side with my full-bell-and-whistles Emacs, without this change:

![image](https://user-images.githubusercontent.com/54230/101051953-5ac9da80-3586-11eb-9536-c4f69fd85174.png)

And with the change:

![image](https://user-images.githubusercontent.com/54230/101052169-995f9500-3586-11eb-8945-466974fce099.png)

Reply:
Did you test with your own configuration around the theme or using emacs -q ?

Reply:
What is the size of the default face in your configuration (without nano) and how do you set it?

Reply:
Some [documentation](https://www.gnu.org/software/emacs/manual/html_node/emacs/Fonts.html) on emacs fonts. From what I read, I think the line is correct so something must be interfering.

Reply:
In my non-nano configuration, default font height is set to 139 (in 1/10pts, so 13.9pts), through `(custom-set-faces)`. Relevant part looks like:
`'(default ((t (:inherit nil :stipple nil :background "#0D0D0D" :foreground "#DCDCCC" :inverse-video nil :box nil :strike-through nil :overline nil :underline nil :slant normal :weight normal :height 139 :width normal :foundry "PfEd" :family "Terminus (TTF)"))))`

What you can see on the right side is always `\emacs -q -l nano.el -dark` (with and without fix applied).

`(describe-face 'default)` in my fullblown Emacs returns:
~~~~
           Family: Terminus (TTF)
          Foundry: PfEd
            Width: normal
           Height: 139
           Weight: normal
            Slant: normal
       Foreground: #DCDCCC
DistantForeground: unspecified
       Background: #0D0D0D
        Underline: nil
         Overline: nil
   Strike-through: nil
              Box: nil
          Inverse: nil
          Stipple: nil
             Font: #<font-object -PfEd-Terminus (TTF)-normal-normal-normal-*-37-*-*-*-m-0-iso10646-1>
          Fontset: -PfEd-Terminus (TTF)-normal-normal-normal-*-37-*-*-*-m-0-fontset-auto2
           Extend: nil
          Inherit: nil
~~~~

`(describe-face 'default)` in master branch using `"Roboto Mono:style=Light:size=14"` :
~~~~
           Family: Roboto Mono
          Foundry: GOOG
            Width: normal
           Height: 53
           Weight: normal
            Slant: normal
       Foreground: #37474F
DistantForeground: unspecified
       Background: #FFFFFF
        Underline: nil
         Overline: nil
   Strike-through: nil
              Box: nil
          Inverse: nil
          Stipple: nil
             Font: #<font-object -GOOG-Roboto Mono-normal-normal-normal-*-14-*-*-*-m-0-iso10646-1>
          Fontset: -GOOG-Roboto Mono-normal-normal-normal-*-14-*-*-*-m-0-fontset-auto1
           Extend: nil
          Inherit: nil
~~~~

`(describe-face 'default)` using my patched nano config:
~~~~
           Family: Roboto Mono
          Foundry: GOOG
            Width: normal
           Height: 120
           Weight: normal
            Slant: normal
       Foreground: black
DistantForeground: unspecified
       Background: white
        Underline: nil
         Overline: nil
   Strike-through: nil
              Box: nil
          Inverse: nil
          Stipple: nil
             Font: #<font-object -GOOG-Roboto Mono-normal-normal-normal-*-32-*-*-*-m-0-iso10646-1>
          Fontset: -GOOG-Roboto Mono-normal-normal-normal-*-32-*-*-*-m-0-fontset-auto1
           Extend: nil
          Inherit: nil
~~~~

Reply:
So .. saying size=14 appears to result into selecting `-GOOG-Roboto Mono-normal-normal-normal-*-14-*-*-*-m-0-fontset-auto1` with `Height: 53` and omitting it results into selecting `-GOOG-Roboto Mono-normal-normal-normal-*-32-*-*-*-m-0-iso10646-1` with `Height: 120`.

This could be some weird fontconfig related issue - the interfering thing here would therefore be my chosen way of installing Roboto Mono. I'll look into it more.

Reply:
Can you put back the original line such that I an merge the PR and we can investigate the problem in a new issue?

Reply:
I did that for you and also opened a new issue with all relevant info.